sbt-plugins/scalahost/src/sbt-test/sbt-scalahost/helloworld/test

```diff
@@ -0,0 +1,5 @@
+# compile should not affect scalafix
+> dumpsemanticdb
+# TODO(olafur) verify that 2.11 and 2.12 got dumped
```

Comment:
I prefer not to block this PR by this bug. My personal use-case for this sbt plugin is to use #666, which I hope to bundle into a PR tomorrow.